node-it/src/test/scala/com/wavesplatform/it/api/AsyncGrpcApi.scala

```diff
@@ -95,9 +99,29 @@ object AsyncGrpcApi {
 
       script match {
         case Left(_) => transactions.broadcast(SignedTransaction.of(Some(unsigned), Seq(ByteString.EMPTY)))
-        case _ =>
-          val proofs =
-            crypto.sign(source.privateKey, PBTransactions.vanilla(SignedTransaction(Some(unsigned)), unsafe = true).explicitGet().bodyBytes())
+        case Right(sc) =>
+          val baseBytes = Bytes.concat(
+            source.publicKey.arr,
+            Deser.serializeArrayWithLength(name.getBytes),
+            Deser.serializeArrayWithLength(description.getBytes),
+            Longs.toByteArray(quantity),
+            Array(decimals.toByte),
+            Deser.serializeBoolean(reissuable),
+            Longs.toByteArray(fee),
+            Longs.toByteArray(unsigned.timestamp)
+          )
+
+          val bodyBytes = version match {
+            case TxVersion.V1 => Bytes.concat(Array(IssueTransaction.typeId), baseBytes)
+            case TxVersion.V2 =>
+              Bytes.concat(Array(IssueTransaction.typeId, version.toByte, chainId), baseBytes, Deser.serializeOptionOfArrayWithLength(sc)(_.bytes().arr))
+            case _ =>
+              val data = IssueTransactionData(name, description, quantity, decimals, reissuable, toPBScript(script))
+              val tx = PBTransactions.create(source.publicKey, chainId, fee, Waves, unsigned.timestamp, version, Proofs.empty, Data.Issue(data)).getTransaction
+              PBUtils.encodeDeterministic(tx)
+          }
+
+          val proofs = crypto.sign(source.privateKey, bodyBytes)
```

Comment:
No 
We must be able to test broadcasting issue transaction with incorrect data (e.g. decimals or amount here https://github.com/wavesplatform/Waves/blob/refined_types/node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala#L156 )
`com.wavesplatform.transaction.serialization.impl.IssueTxSerializer.toBytes` function needs `IssueTransaction` instance, which we can't create with invalid data because of refined types.

Reply:
Done